corehq/apps/accounting/async_handlers.py

```diff
@@ -159,9 +160,9 @@ class Select2BillingInfoHandler(BaseSelect2AsyncHandler):
     @property
     def country_response(self):
         from django_countries.data import COUNTRIES
-        countries = sorted(COUNTRIES.items(), key=lambda x: x[1].encode('utf-8'))
+        countries = sorted(list(COUNTRIES.items()), key=lambda x: x[1].encode('utf-8'))
         if self.search_string:
-            return filter(lambda x: x[1].lower().startswith(self.search_string.lower()), countries)
+            return [x for x in countries if x[1].lower().startswith(self.search_string.lower())]
```

Comment:
nit: I think this would run `self.search_string.lower()` for each country. could pull that out to a variable and make this a little easier to read. Similar for the others in this file